IssueTracker/DataAccess/Models/Issue.cs

```diff
@@ -22,11 +16,12 @@ public class Issue
         public int StatusId { get; set; }
         public int RoleId { get; set; }
         public bool IsDeleted { get; set; }
-        public IssueType? IssueType { get; set; }
-        public Priority? Priority { get; set; }
-        public Status? Status { get; set; }
-        public Role? Role { get; set; }
-        public Project? Project { get; set; }
+        public IssueType? IssueType { get; set; } = new IssueType();
```

Comment:
doesn't look right. why is this marked optional and then instantiated?

Reply:
public IssueType IssueType { get; set; } = new IssueType();
        public Priority Priority { get; set; } = new Priority();
        public Status Status { get; set; } = new Status();
        public Role Role { get; set; } = new Role();
        public Project Project { get; set; } = new Project();
        public User User { get; set; } = new User();

Reply:
why is instantiation needed here?